problems/2.functions.js

```diff
@@ -2,36 +2,40 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function greeting(name) {
```

Comment:
If it takes in a argument, it will need probably need to use it. If not, recognize that as a code smell.